forms-flow-web/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Base image
-FROM node:14.17.6-alpine as build-stage
+FROM node:14.17.6 as build-stage
```

Comment:
Why changing alpine version ? Alpine will be more smaller image

Reply:
in this pr, I have attached a snapshot of the error. because the Alpine version doesn't contain all functionality